amazon-kinesis-client/pom.xml

```diff
@@ -207,6 +207,10 @@
               <name>sqlite4java.library.path</name>
               <value>${sqlite4java.libpath}</value>
             </property>
+            <property>
+              <name>credentials</name>
+              <value>${credentials}</value>
```

Comment:
I added some clarification to the README about how this parameter would be used. You can optionally run ` mvn -Dit.test=TESTNAME -Dcredentials=IAM_USER_NAME verify` to override the default IAM user. If this parameter is not provided in the maven command, it will use the default credentials of the test environment where the code is being run.